src/core/ext/filters/deadline/deadline_filter.cc

```diff
@@ -384,27 +381,15 @@ const grpc_channel_filter grpc_server_deadline_filter = {
     "deadline",
 };
 
-bool grpc_deadline_checking_enabled(
```

Comment:
Please also remove the declaration of this function from deadline_filter.h.

Reply:
done